### DISCUSSION TITLE
Relations import via JSONL

### DISCUSSION THREAD

**Discussion by karikara95** [doccano/doccano#1809] [Ideas]
Hello, 

For the Seq labeling task, is there a way to import relations in JSONL if the entities' spans are defined?

Thank you

**Comment by leonardlin** [comment]
yes, the current jsonl importer supports relationships.
You can create a few annotations in doccano with relationships.
Export it and you see the dataformat.

**Comment by david-engelmann** [comment]
You'll need to select the correct option from the export dataset section. There are two options for export, if you want to see how the relations will need to be upload you can export as JSONL(Relation).

![image](https://user-images.githubusercontent.com/81196676/199324354-f971331f-bcbe-4703-9c19-439ce2f51bb8.png)

The results of the export will need to be altered, the id field should be removed.

If you'd like to view a sample of an input file, I've included it below

```
{
    "text": "Google was founded on September 4, 1998, by Larry Page and Sergey Brin.",
    "entities": [
        {
            "id": 0,
            "start_offset": 0,
            "end_offset": 6,
            "label": "ORG"
        },
        {
            "id": 1,
            "start_offset": 22,
            "end_offset": 39,
            "label": "DATE"
        },
        {
            "id": 2,
            "start_offset": 44,
            "end_offset": 54,
            "label": "PERSON"
        },
        {
            "id": 3,
            "start_offset": 59,
            "end_offset": 70,
            "label": "PERSON"
        }
    ],
    "relations": [
        {
            "id": 0,
            "from_id": 0,
            "to_id": 1,
            "type": "foundedAt"
        },
        {
            "id": 1,
            "from_id": 0,
            "to_id": 2,
            "type": "foundedBy"
        },
        {
            "id": 2,
            "from_id": 0,
            "to_id": 3,
            "type": "foundedBy"
        }
    ]
}
```

**Reply by david-engelmann** [reply]
Also, there are some current bugs with the current import and export process with relation labels, you can follow along with progress resolving them in [this issue](https://github.com/doccano/doccano/issues/2039)
